src/bin/hq.rs

```diff
@@ -135,6 +135,8 @@ struct JobDetailOpts {
 #[derive(Clap)]
 #[clap(setting = clap::AppSettings::ColoredHelp)]
 struct SubmitOpts {
+    #[clap(long)]
```

Comment:
Please add help for the option. It's enough to add a doc comment above the attribute (see other `*Opts` structs.